src/ai/backend/common/redis.py

```diff
@@ -264,7 +269,8 @@ async def execute(
             aioredis.sentinel.SlaveNotFoundError,
             aioredis.exceptions.ReadOnlyError,
             ConnectionResetError,
-        ):
+        ) as e:
+            log.error(e)
```

Comment:
Our logger has method specialized for logging exceptions: `log.exception()`. By calling this without any parameter, logger will automatically retrieve current stack trace information and print it to destination. Feel free to search some common usages on other codes (agent, manager, ...) and please rewrite this.